polaris-core/src/main/java/org/apache/polaris/core/auth/CoreAuthorizer.java

```diff
@@ -529,27 +542,26 @@ public void authorizeOrThrow(
     }
   }
 
-  /**
-   * Based on the required target/targetParent/secondary/secondaryParent privileges mapped from
-   * {@code authzOp}, determines whether the caller's set of activatedGranteeIds is authorized for
-   * the operation.
-   */
-  public boolean isAuthorized(
-      @Nonnull AuthenticatedPolarisPrincipal authenticatedPolarisPrincipal,
-      @Nonnull Set<PolarisBaseEntity> activatedEntities,
-      @Nonnull PolarisAuthorizableOperation authzOp,
-      @Nullable PolarisResolvedPathWrapper target,
-      @Nullable PolarisResolvedPathWrapper secondary) {
-    return isAuthorized(
-        authenticatedPolarisPrincipal,
-        activatedEntities,
-        authzOp,
-        target == null ? null : List.of(target),
-        secondary == null ? null : List.of(secondary));
+  private boolean isSelfReset(
```

Comment:
Okay, it took me a while to realize that this comes from `PolarisAdminService.authorizeBasicTopLevelEntityOperationOrThrow()`. I agree with the change.

Reply:
I'll add a comment to clarity

Reply:
updated